package.json

```diff
@@ -9,8 +10,8 @@
     "LICENSE"
   ],
   "browser": {
-    "pnpapi": false,
-    "process": "./lib/util/process-browser.js"
+    "process": "./lib/util/process-browser.js",
+    "module": "./lib/util/module-browser.js"
```

Comment:
Why we `module` here?

Reply:
@merceyz I'm assuming we've specified the `module` main field here for resolvers that can utilize ESM? Do we need that though for `enhanced-resolve` and more specifically for this feature to make it through?

Reply:
For the same reason `process` is there, it's not supported in browsers.